supabase/tests/65_test_github_user_id.sql

```
@@ -0,0 +1,42 @@
+  (
+    SELECT is_nullable
+    FROM information_schema.columns
+    WHERE table_schema = 'public'
+      AND table_name = 'users'
+      AND column_name = 'github_id'
+  ),
+  'YES',
+  'users.github_id is nullable'
+);
+
+SELECT * FROM finish();
```

SQLFluff (4.2.2)

[error] Query produces an unknown number of result columns.

(AM04)